lib/edvs.js

```diff
@@ -0,0 +1,92 @@
+/*!
+ * Copyright (c) 2020 Digital Bazaar, Inc. All rights reserved.
+ */
+'use strict';
+
+const bedrock = require('bedrock');
+const brHttpsAgent = require('bedrock-https-agent');
+const {EdvClient} = require('edv-client');
+const keyResolver = require('./keyResolver');
+const kms = require('./kms');
+
+// load config defaults
+require('./config');
```

Comment:
I notice that this is getting required several times, can it just be called once in index.js or similar?

Reply:
Theoretically... yes. If we can make the assumption that these modules are accessed through a single entry point, then we can place the require config call in a single location.